lib/indexer/links_lookup.rb

```diff
@@ -23,12 +24,19 @@ def prepare_tags(doc_hash)
       content_id = find_content_id(doc_hash)
       return doc_hash unless content_id
 
+      content_item = {
```

Comment:
Hmm, maybe this is related to https://github.com/alphagov/email-alert-service/pull/67#discussion_r106368769. Could we somehow avoid doing this just for the sake of the gem? Seems we're fighting the API here.